io/subsystem/src/main/java/org/wildfly/extension/io/BufferPoolResourceDefinition.java

```diff
@@ -126,6 +139,7 @@ private BufferPoolResourceDefinition() {
     @Override
     public void registerCapabilities(ManagementResourceRegistration resourceRegistration) {
         resourceRegistration.registerCapability(IO_POOL_RUNTIME_CAPABILITY);
+        resourceRegistration.registerCapability(IO_BYTE_BUFFER_POOL_RUNTIME_CAPABILITY);
```

Comment:
this does not prevent to merge this PR but this capability needs to be documented in https://github.com/wildfly/wildfly-capabilities